core/src/main/java/hudson/model/Run.java

```diff
@@ -730,9 +730,12 @@ public String getDescription() {
      * Gets the string that says how long the build took to run.
      */
     public @Nonnull String getDurationString() {
-        if(isBuilding())
+        if (hasntStartedYet()) {
```

Comment:
I was not responsible for the name but "hasn't" it is a short form of "has not".